spring-jasper-example/src/main/resources/application-client-list-local.yml

```
@@ -4,4 +4,4 @@ server:
 spring:
 
   application:
-    name: spring-jasper-exampleclient-list-local
\ No newline at end of file
+    name: spring-jasper-example-client-list-local
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)